src/main/kotlin/me/ezra_home/retail_software_solution/business/util/mapper/userinfo/FruitMapper.kt

```diff
@@ -0,0 +1,38 @@
+package me.ezra_home.retail_software_solution.business.mapper
```

Comment:
The userinfo package is meant for classes that relate to user info. Fruit is not related to user and cannot be in this package